modules/mod_auth_unix.c

```diff
@@ -1002,6 +1004,9 @@ MODRET pw_check(cmd_rec *cmd) {
         "AIX authenticate result: %d (msg '%.100s')", res, msg);
 
     } while (reenter != 0);
+    if (res == 0) {
+      login_succeeded(cmd->tmp_pool, user);
```

Comment:
Rather than reaching naughtily into the `mod_auth` symbols, I've instead used a little code duplication in https://github.com/proftpd/proftpd/pull/966, to achieve the same functionality here.